### DISCUSSION TITLE
Smithy and Playframework

### DISCUSSION THREAD

**Discussion by jona7o** [disneystreaming/smithy4s#525] [Show and tell]
Hi,

thanks for the great work on smithy4s. Awesome! 
We thought that it is a great opportunity for us to build another abstraction layer for the playframework as we used it in a lot of projects even when there are some difficulties regarding the akka licensing right now. 

Based on smithy4s we created [smithy4play](https://github.com/innFactory/smithy4play). With smithy4play you can generate everything you need for the routing and for your test scenarios. With the API-first approach it can replace the classical routes.conf file and the default routing without writing much boilerplate code. In the next days we will also add a minimal example and write a blog post on meidum. 

Please feel free to give us feedback or ideas for the package. If "[pekko](https://github.com/mdedetrich/akka-apache-project/discussions/16)" gets an apache project we will also release a smithy4pekko project for "pekko-http" (akka-http). 

https://github.com/innFactory/smithy4play

**Comment by Baccata** [comment]
Hah, happy to see some people are taking it on :) 

Internally, at work, we actually have akka-http and play integration, we haven't released them to avoid increasing the (already high) maintenance burden inherent to open-source software. It makes me **really** happy that other people/orgs are taking enough of an interest in smithy4s to build third-party integrations 😄 

It makes me all the happier that you guys didn't ask for help, as Smithy4s is not the easiest codebase to look at. 

Kudos to you and your team !

**Comment by kubukoz** [comment]
@Baccata think we should add a link for this somewhere in the docs/FAQ?

**Reply by Baccata** [reply]
Sure yeah
